specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/stable/2020-05-15/timeseriesinsights.json

```diff
@@ -0,0 +1,2549 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "TimeSeriesInsightsClient",
+    "description": "Time Series Insights client",
+    "version": "2020-05-15"
```

Comment:
>version [](start = 5, length = 7)

Is there a previous API version that was approved for Time Series Insights that I can compare this with?

Reply:
This one should be almost identical to the [2018-08-15-preview](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/preview/2018-08-15-preview/timeseriesinsights.json) public preview api-version

Reply:
Thanks that helps a lot.

---
In reply to: [400576000](https://github.com/Azure/azure-rest-api-specs/pull/8791#discussion_r400576000) [](ancestors = 400576000)